AGAMI/Sources/Coordinator/PlakeCoordinator.swift

```diff
@@ -14,7 +14,40 @@ enum PlakeRoute: Hashable {
     
     case newPlakeView
     case searchShazamingView
-    case searchWritingView
+    case searchWritingView(viewModel: SearchStartViewModel)
+    case cameraView(viewModel: SearchWritingViewModel)
+    
+    // Hashable 직접 구현
+    func hash(into hasher: inout Hasher) {
+        switch self {
+        case .listView:
+            hasher.combine("listView")
+        case .playlistView:
+            hasher.combine("playlistView")
+        case .newPlakeView:
+            hasher.combine("newPlakeView")
+        case .searchShazamingView:
+            hasher.combine("searchShazamingView")
+        case .searchWritingView:
+            hasher.combine("searchWritingView")
+        case .cameraView:
+            hasher.combine("cameraView")
+        }
+    }
+    
+    static func == (lhs: PlakeRoute, rhs: PlakeRoute) -> Bool {
+        switch (lhs, rhs) {
+        case (.listView, .listView),
+            (.playlistView, .playlistView),
+            (.newPlakeView, .newPlakeView),
+            (.searchShazamingView, .searchShazamingView),
+            (.searchWritingView, .searchWritingView),
+            (.cameraView, .cameraView):
+            return true
+        default:
+            return false
+        }
+    }
```

Comment:
id 만들고 id를 비교하는 방법은 어떨까요? 
```suggestion
    var id: String {
        switch self {
        ...
        }
    }
    func hash(into hasher: inout Hasher) {
        hasher.combine(id)
    }
    
    static func == (lhs: PlakeRoute, rhs: PlakeRoute) -> Bool {
        lhs.id == rhs.id
    }
```

Reply:
오 좋아요! 반영하겠습니당